packages/terra-modal-manager/tests/wdio/modal-manager-spec.js

```diff
@@ -492,7 +500,7 @@ Terra.describeViewports('ModalManager - Disclosure Accessory', ['large'], () =>
 
     it('renders each disclosed component in a disclosure container', () => {
       expect(browser.isExisting('[data-disclosure-container="true"]')).to.be.true;
-      Terra.validates.element('nested disclosure', { selector });
+      Terra.validates.element('nested disclosure', { selector, axeRules: ignoredA11y });
```

Comment:
This may be unrelated but recently I noticed that this [modal manager test page](https://engineering.cerner.com/terra-framework/tests/terra-modal-manager/modal-manager/modal-manager-integration) has a select field that does not seem to do anything. I'm curious, what is it for?

Reply:
This is/was used previously to validate that the Select hookshot implementation worked as expected within Modals. The Select dropdown is using hookshot to position itself. The z-layer ordering and positioning changes are really important to validate are working as intended.